flare/base/net/endpoint.cc

```diff
@@ -60,9 +56,7 @@ std::string SockAddrToString(const sockaddr* addr) {
       FLARE_CHECK(
           inet_ntop(af, &p->sin6_addr, result.data() + 1, result.size()));
       auto ptr = result.data() + result.find('\x00');
-      *ptr++ = ']';
-      *ptr++ = ':';
-      snprintf(ptr, kPortChars, "%d", ntohs(p->sin6_port));
+      FormatTo(ptr, "]:{}", ntohs(p->sin6_port));
```

Comment:
Without a way to specify buffer size for `ptr`, this feels dangerous to me. Maybe we can instead use `inet_ntop` to format IP to a local char buffer, and then use `format` to generate the final `[ip]:port` string?

Reply:
I agree with you.

Reply:
Still need to keep `FormatTo`?

Reply:
I’m okay with it.